<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LocalWorldController</class>
 <widget class="QWidget" name="LocalWorldController">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,1">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QGraphicsView" name="graphicsView"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>1</number>
       </property>
       <item>
        <widget class="QComboBox" name="comboBox"/>
       </item>
       <item>
        <widget class="QTreeWidget" name="checkTree">
         <column>
          <property name="text">
           <string>Available Graphical Elements</string>
          </property>
         </column>
         <item>
          <property name="text">
           <string>(Un) Check all</string>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
          <item>
           <property name="text">
            <string>Estimated Robot Pose</string>
           </property>
           <property name="checkState">
            <enum>Checked</enum>
           </property>
          </item>
          <item>	
           <property name="text">
            <string>Robot Pose Uncertainty</string>
           </property>
           <property name="checkState">
            <enum>Checked</enum>	
           </property>
          </item>
          <item>	
           <property name="text">
            <string>Robot Pose Hypothesis</string>
           </property>
           <property name="checkState">
            <enum>Checked</enum>	
           </property>
          </item>
          <item>
           <property name="text">
            <string>Odometry</string>
           </property>
           <property name="checkState">
            <enum>Checked</enum>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Robot Trace</string>
           </property>
           <property name="checkState">
            <enum>Checked</enum>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Estimated Ball Position</string>
           </property>
           <property name="checkState">
            <enum>Checked</enum>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Ball Observation</string>
           </property>
           <property name="checkState">
            <enum>Checked</enum>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Landmark Observations</string>
           </property>
           <property name="checkState">
            <enum>Checked</enum>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Localization Particles</string>
           </property>
           <property name="checkState">
            <enum>Checked</enum>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Robot View Field Projection</string>
           </property>
           <property name="checkState">
            <enum>Checked</enum>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Walk Commands</string>
           </property>
           <property name="checkState">
            <enum>Checked</enum>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Estimated Teammate Positions</string>
           </property>
           <property name="checkState">
            <enum>Checked</enum>
           </property>     
          </item>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
